platform/lib/platform/material/media_search.ex

```diff
@@ -246,6 +284,106 @@ defmodule Platform.Material.MediaSearch do
     end
   end
 
+  defp apply_query_component(queryable, changeset, arbitrary_key) do
+    # Filters for project attributes
+    with rel_changes <- Map.get(changeset.changes, arbitrary_key),
+         false <- is_nil(rel_changes),
```

Comment:
Nit: could do these in a single match; i.e., `rel_changes when not is_nil(rel_changes) <- Map.get(changeset.changes, arbitrary_key)`.

Reply:
Interesting! Did not know of this syntax